package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "GitHub.Expandinizr",
```

Comment:
I seem to recall there being an issue with upper-case package names? Or was that another issue ?